### ISSUE TITLE
Settings does not contain tabs after standard installation on clean system

### ISSUE THREAD

**Issue by douwevt** [spotweb/spotweb#380]
I just installed Spotweb on a Raspberry Pi 3 Model B+ (Debian 9) clean installation. After installation, and logging in as admin, in the settings page, the tabs are missing.

**Comment by Sweepr** [comment]
Which php version?

**Comment by douwevt** [comment]
Thank you for your response! I used the version that is automatically installed during the installation of spotweb, being version 7.0.

**Comment by douwevt** [comment]
What I tried was removing spotweb with apt-get remove spotweb, and copied the latest spotweb version to /var/www/html (the spotweb that is installed by apt-get install spotweb is in /usr/share/spotweb), and then I got other errors, like unable to create directory on the localhost/spotweb/install.php page but the tabs were there. So it seems to be an issue with the distribution of spotweb in Debian Stretch. Not so much the underlaying software stack. What I also noticed in this distribution is that f.i. install.php is not there.
